Samples/MediaCaptureSampleExtendedControl/KsStructures.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+#include <ks.h>
+#include <ksmedia.h>
+
+//
+// The KS Structures needed for this sample are not defined for WinRT. We redefine them here, copying their definition
```

Comment:
>WinRT [](start = 64, length = 5)

if this a public defined guid , we should be able to just include this here since this is just cpp code.

Reply:
KSMedia.h has several #ifdefs that prevent this inclusion. Looking at changing that for new SDKs, but for existing usage, you have to copy manually.